### ISSUE TITLE
Consider using local files instead of unpkg.com

### ISSUE THREAD

**Issue by animaux** [acoustep/craft-easymde#6]
I’ve experienced this with many sites and plugins in recent years: stuff breaks because of changes or shutdown in external services. So for long-term-support I’d expect the use of local assets to be a more futureproof solution.

**Comment by acoustep** [comment]
I agree I prefer local files, if I remember correctly I had issues doing this previously which is why I went the external route. I'll review this with Craft 4.
